src/components/BorrowerProfile/SocialShareButton.vue

```diff
@@ -0,0 +1,267 @@
+			}
+			return '';
+		},
+		utmContent() {
+			if (this.lender?.public && this.lender?.inviterName) return this.lender?.inviterName;
+			return 'anonymous';
+		},
+		shareLink() {
+			const base = `https://${this.$appConfig.host}`;
+			if (this.loan.id) {
+				return `${base}/invitedby/${this.lender.inviterName}/for/${this.loan.id}?utm_content=${this.utmContent}`; // eslint-disable-line max-len
+			}
+			return `${base}?utm_content=${this.utmContent}`;
+		},
+		facebookShareUrl() {
+			const pageUrl = `https://${this.$appConfig.host}${this.$route.path}`;
+			return getFullUrl('https://www.facebook.com/dialog/share', {
+				app_id: this.$appConfig.fbApplicationId,
+				display: 'page',
+				href: `${this.shareLink}&utm_source=facebook.com&utm_medium=social&utm_campaign=social_share_bp_pfp`,
+				redirect_uri: `${pageUrl}`,
+				quote: this.shareMessage,
+			});
+		},
+		linkedInShareUrl() {
+			return getFullUrl('https://www.linkedin.com/shareArticle', {
+				mini: 'true',
+				source: `https://${this.$appConfig.host}`,
+				summary: this.shareMessage.substring(0, 256),
+				title: `A loan for ${this.loan.name}`,
+				url: `${this.shareLink}&utm_source=linkedin.com&utm_medium=social&utm_campaign=social_share_bp_pfp`
+			});
+		},
+		twitterShareUrl() {
+			return getFullUrl('https://twitter.com/intent/tweet', {
+				text: this.shareMessage,
+				url: `${this.shareLink}&utm_source=t.co&utm_medium=social&utm_campaign=social_share_bp_pfp`,
+				via: 'Kiva',
+			});
+		},
+	},
+	methods: {
+		handleFacebookResponse() {
+			// Check for the route hash that facebook adds to the request
+			if (this.$route.hash === '#_=_') {
+				// Check for an error
+				const { error_code: code, error_message: message } = this.$route.query;
+				if (code) {
+					// The 4201 error code means the user pressed 'Cancel', so can be ignored
+					if (code !== '4201') {
+						this.$showTipMsg(`There was a problem sharing to Facebook: ${message}`, 'warning');
+					}
+				} else {
+					this.$showTipMsg('Thanks for sharing to Facebook!');
+				}
+			}
+		},
+		async copyLink() {
+			const url = `${this.shareLink}&utm_source=social_share_link&utm_campaign=social_share_bp_pfp`;
+			try {
+				await clipboardCopy(url);
+				this.copyStatus = {
+					disabled: true,
+					text: 'Copied!'
+				};
+			} catch (err) {
+				this.copyStatus = {
+					disabled: true,
+					text: 'Error'
+				};
+			} finally {
+				setTimeout(() => {
+					this.copyStatus = {
+						disabled: false,
+						text: 'Copy Link'
+					};
+				}, 500);
+			}
+		}
+	},
+	mounted() {
+		this.handleFacebookResponse();
+	},
+};
+</script>
+
+<style lang="postcss" scoped>
+.social-button {
+	@apply tw-mb-2.5 tw-w-fit;
+}
+
+.social-button__icon {
+	@apply tw-w-5 tw-h-5 tw-pointer-events-none tw-inline-block tw-align-middle;
+}
+</style>
+
+<style lang="scss" scoped>
```

Comment:
Do we want to replace the `scss` with `postcss`, or are we still using SCSS in the `ui` project?

Reply:
It's worth having a look to see if postcss will cover all of the existing usages of scss across the Ui repo...

Reply:
using `lang="postcss"` will allow you to @apply tailwind classes. It does NOT allow nested CSS, while `lang="scss"` allows nesting and variables, but it does not allow you to @apply tailwind classes

Reply:
Right right, cheers to multiple style blocks!